### DISCUSSION TITLE
Home Assistant | How to integrate Frigate with Home Assistant when each is on their own server

### DISCUSSION THREAD

**Discussion by vexersa** [blakeblackshear/frigate#12057] [General]
I hope that this hasn't been answered before, I did search for some time in the discussions but couldn't seem to find anything similar. 

I have Proxmox host with two containers: HA and Frigate. 

I would like to integrate Frigate into HA so that the camera entities are found and are able to be used by Frigate Lovelace Card, etc. 

I have added the Frigate Proxy add-on which works and connects to my Frigate instance well. 

I am however a little unguided on how to integrate Frigate in a way that enables the use of the respective detected entities; cameras, sensors (events?), etc. 

I assume that the ordinary way of getting this to work is to use the Frigate add-on for HA which runs a Frigate instance within HA and integrates the cameras as entities etc, but how would I replicate this functionality in my setup?

Any guidance would be greatly appreciated.

**Comment by a-lunatic** [comment]
[Frigate Home Assistant Integration](https://github.com/blakeblackshear/frigate-hass-integration)

**Comment by vexersa** [comment]
@a-lunatic 

Thanks, though I am not sure how to configure the seperate instance as the documentation does not seem complete in this regard. 

There is no prompt in the add-on's UI:
![image](https://github.com/blakeblackshear/frigate/assets/7994227/ad434ba2-1da7-495d-89c4-e25cd9618d69)

In the documentation, I'm not sure how to configure this?:
![image](https://github.com/blakeblackshear/frigate/assets/7994227/cb666ddc-397d-47cc-b45a-55885f100b81)

**Reply by a-lunatic** [reply]
It's not an addon you need to add it with HACS or manual installation

> HACS -> Integrations -> Explore & Add Repositories -> Frigate

Notes:

 >   HACS does not "configure" the integration for you. You must go to Configuration > Integrations and add Frigate after installing via HACS.
    The mqtt integration must be installed and configured in order for the Frigate integration to work. As manual configuration is required for the mqtt setup, this cannot happen automatically.
>
>For manual installation for advanced users, copy custom_components/frigate to your custom_components folder in Home Assistant.

**Reply by vexersa** [reply]
Thanks for the help. 

I guess my eyes just weren't reading right. It looks to be working now :)
